src/CalcManager/UnitConverter.cpp

```diff
@@ -505,142 +498,144 @@ void UnitConverter::SendCommand(Command command)
 {
     if (CheckLoad())
     {
-        //TODO: Localization of characters
-        bool clearFront = false;
-        if (m_currentDisplay == L"0")
+        return;
```

Comment:
This change is incorrect.  You didn't invert the conditional check above.

Reply:
Yes indeed.
Fixed